src/pages/Components/Notification/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Notification'
+order: 4
+---
+
+> Notifications are useful to bring user's attention about an information available in four types of alert. A notification often appears as a result of a user action.
```

Comment:
can we try to put it that way : 
```Notifications are used to bring user's attention on an information he needs to know. There are four types of alerts. A notification is usually the result of an action made by the user.```

Reply:
Ok, done. I just changed the word "alert" to "notification", Gael well observed that we've been using this word to describe the "notification pattern" itself and it may be confusing for the user.